src/core/chatController.spec.js

```diff
@@ -42,10 +42,10 @@ describe("ChatController", () => {
     let endResponse;
 
     function getChatController(shouldSendMessageReceipts = true) {
-        GlobalConfig.update({
-            features: shouldSendMessageReceipts ? [FEATURES.MESSAGE_RECEIPTS_ENABLED] : [],
-            throttleTime: 1000
-        });
+        if (!shouldSendMessageReceipts) {
+            GlobalConfig.removeFeatureFlag(FEATURES.MESSAGE_RECEIPTS_ENABLED);
+        }
+        GlobalConfig.updateThrottleTime(1000);
```

Comment:
where does this 1000ms throttle time come from?

Reply:
Looks like this was just copy/pasted from the internal code. No test coverage so it looks like the behavior won't change it either way

> The README and code will default to 5000 [[ref](https://github.com/amazon-connect/amazon-connect-chatjs/blob/cef17c302e2c83cb7f31c6125e9e53b34f354bbc/src/globalConfig.js#L44)]